### DISCUSSION TITLE
KeepAlive logging suddenly enabled with no change to logging setting

### DISCUSSION THREAD

**Discussion by lupomikti** [Tampermonkey/tampermonkey#1974] [Q&A] [answered]
Hello, I wanted to ask here before assuming this is an issue. After the update to 5.0.1 (Firefox), on the one particular site I frequently use and edit scripts for the console is being filled with messages of the form `injected: keepAlive: got message` followed by the object even though my logging setting has not been changed from Errors. Is this intentional? Why is it only happening on some sites and not all I have scripts running on (well, more accurately for me, just one site, but I assume it could be more)? Is there a way to disable this currently?

Edit: I have managed to narrow the scope of this. The messages are happening only when specific scripts are active. The common factor between each of these scripts is that they all use `GM_getValue` and `GM_setValue`. To confirm this I edited them to comment out all usage, the message logging still occurred. It only stopped once I removed the `@grant`s for the two functions. As this was not happening before the update to 5.0.1 I can only assume this is an unintentional bug wherein the logging for this not respecting the log level setting.

**Comment by derjanb** [comment, marked as answer]
> Is this intentional?

No, will be fixed soon. Sorry.

**Comment by Kesig777** [comment]
Yrack down Nd down supports febice top only
